creator-node/src/services/stateMachineManager/stateReconciliation/updateReplicaSet.jobProcessor.ts

```diff
@@ -695,18 +695,38 @@ const _issueUpdateReplicaSetOp = async (
         return response
       }
 
-      await audiusLibs.contracts.UserReplicaSetManagerClient._updateReplicaSet(
-        userId,
-        newReplicaSetSPIds[0], // new primary
-        newReplicaSetSPIds.slice(1), // [new secondary1, new secondary2]
-        // This defaulting logic is for the edge case when an SP deregistered and can't be fetched from our mapping, so we use the SP ID from the user's old replica set queried from the chain
-        oldPrimarySpId || chainPrimarySpId,
-        [
-          oldSecondary1SpId || chainSecondarySpIds?.[0],
-          oldSecondary2SpId || chainSecondarySpIds?.[1]
-        ]
+      logger.info(
+        `[_issueUpdateReplicaSetOp] updating replica set now ${
+          Date.now() - startTimeMs
+        }ms for userId=${userId} wallet=${wallet}`
       )
 
+      const { blocknumber } =
+        await audiusLibs.User.updateEntityManagerReplicaSet({
+          userId,
+          primary: newReplicaSetSPIds[0], // new primary
+          secondaries: newReplicaSetSPIds.slice(1), // [new secondary1, new secondary2]
+          // This defaulting logic is for the edge case when an SP deregistered and can't be fetched from our mapping, so we use the SP ID from the user's old replica set queried from the chain
+          oldPrimary: oldPrimarySpId || chainPrimarySpId,
+          oldSecondaries: [
+            oldSecondary1SpId || chainSecondarySpIds?.[0],
+            oldSecondary2SpId || chainSecondarySpIds?.[1]
+          ]
+        })
+
+      // Wait for blockhash/blocknumber to be indexed
```

Comment:
what happens when update call fails to go through - will it 500?
can discuss this flow a bit on call